.vsts.pipelines/steps/run-bootstrap.yml

```diff
@@ -0,0 +1,234 @@
+# Delete some files from the default image since we're tight on disk space.
+# List is from https://github.com/microsoft/azure-pipelines-image-generation/pull/1231#issuecomment-537174621
+
+# Extract the tarball fetched from the previous Offline stage 
+steps:
+- script: |
+      set -ex 
+      df -h 
+      $(docker.run) --network=none $(docker.bst.map) $(docker.bst.work) $(imageName) bash -c '
+      mkdir -p /bst/sb/tarball
+      tar -xvf "/bst/Tarball centos71 Offline/$(tarballName).tar.gz" -C /bst/sb/tarball --no-same-owner --strip-components=1
+      rm -rf "/bst/Tarball centos71 Offline"'
+  displayName: Unpack Tarball
+  condition: always()
+  continueOnError: true
```

Comment:
A lot of these `condition: always(); continueOnError: true` directives seem odd to me.  I think right now this leg would pass even if none of the steps do - is that what we're going for?

Reply:
I get the point. I think we don't want to continue if certain steps fail. Ill rectify that. Thanks! 👍